packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -61,6 +75,10 @@ export default class Sayt extends LitElement {
     this.processSfxSearchboxChange = this.processSfxSearchboxChange.bind(this);
     this.setSearchboxListener = this.setSearchboxListener.bind(this);
     this.handleAutocompleteTermHover = this.handleAutocompleteTermHover.bind(this);
+    this.getDebounce = this.getDebounce.bind(this);
+    this.setDebounce = this.setDebounce.bind(this);
```

Comment:
`get` and `set` aren't that clear to me for this purpose; in particular, `setDebounce` suggests to me that you're setting the `debounce` property, which isn't what it's doing. `setDebounce` can be called `setDebouncedMethods` or something like that, and I suggest later on in this review that we probably don't need `getDebounce`.